### ISSUE TITLE
aria-describedby (JAWS key + Alt + R audible cue)

### ISSUE THREAD

**Issue by devarshipant** [FreedomScientific/VFO-standards-support#98]
### Example:

1. https://codepen.io/devpant/pen/xJgZBO

2. Tab to focusable controls and note how jaws prompts the shortcut "jaws key plus Alt plus R" and then reads the referenced text.

### Expected result

1. Users should only hear the describedby text upon initiating 'JAWS key + Alt + R'. They should be able to ignore the describedby reference.
 
2.  The prompt 'JAWS key + Alt + R' should work similar to aria-controls (JAWS key + Alt + M) where the user decides to move focus.

### Actual result

1. "Press JAWS Key plus Alt plus R" audible cue only works on input controls and static text.

2. In links, the shortcut key combination is not spoken, and JAWS directly speaks the referenced element after speaking the accessible name.

3. [IE11 / JAWS 18 / JAWS 2018]: The prompt 'JAWS key + Alt + R' is broken. On activating the shortcut key combination, the virtual cursor does not move to describedby reference.

4. [Chrome 68 / JAWS 18 / JAWS 2018]: The prompt 'JAWS key + Alt + R' works. Virtual cursor correctly moves to referenced element using the shortcut key combination.

### Additional Information

- 18.0.2531;

 - 2018.1805.33

 
#### Operating System and version

- Win 10 / 64 Bit

#### Browser and version:

- IE11; Chrome 68

**Comment by JAWS-test** [comment]
The problem was fixed in JAWS Version 2020
